synapse/api/room_versions.py

```diff
@@ -116,6 +118,7 @@ class RoomVersions:
         special_case_aliases_auth=True,
         strict_canonicaljson=False,
         limit_notifications_power_levels=False,
+        msc2175_create_events=False,
```

Comment:
It's unfortunate that `create` is a verb here. `msc2175_create_events=False,` reads like "Don't create events" to me.

Only suggestion I have is "msc2175_implicit_room_creator`.

(Don't let the naming block you from landing this, it's good enough as is)

Reply:
That sounds nicer, thanks!